Andrew_Aspect TL Activity

andrewcshannon opened the pull request:
Adding "Andrew_Aspect" .json and .py files for GIS714 Tangible Landscape Activity Spring 2022

andrewcshannon committed:
Adding .json and .py files

Adding .json and .py files for GIS714 Tangible Landscape Activity

andrewcshannon committed:
Merge pull request #1 from andrewcshannon/andrewcshannon-TL

Adding .json and .py files

andrewcshannon committed:
Update after running Black

petrasovaa commented on activities/Andrew_Aspect.py:
```diff
@@ -0,0 +1,51 @@
+        env=env,
+    )
+    # reclassify using rules passed as a string to standard input
+    # 0:22.5:1 means reclassify interval 0 to 22.5 percent of aspect to category 1
+    rules = [
+        "0:22.5:1",
+        "22.5:67.5:1",
+        "67.5:292.5:NULL",
+        "292.5:337.5:1",
+        "337.5:360:1",
+        "*:-9999:NULL",
+    ]
```
Not sure I understand why 22.5 or 67.5? It's not percent, it's degrees. Why not 45?

andrewcshannon replied:
I will correct that terminology in the comment. But, I want to capture all North-facing aspects, including west of northwest (0 to 67.5 degrees) and east of northeast (292.5 to 0 degrees). I see that I can shorten those classes in this rules list, which I will update, as well.

andrewcshannon committed:
Updated reclassification & comment lines 

Update reclassification rules to shorten number of ranges reclassified and elaborate on the reclassification motivation in the commented lines

andrewcshannon committed:
Removed typo

andrewcshannon committed:
Re-ran black
